plugins/in_head/in_head.c

```diff
@@ -434,6 +382,52 @@ static int in_head_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */
+static struct flb_config_map config_map[] = {
+    // lines
```

Comment:
Single line comment should be `/* lines */`.
https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#single-line-comment